src/angularjs/src/app/components/navbar/navbar.directive.js

```diff
@@ -35,7 +31,11 @@
             templateUrl: 'app/components/navbar/navbar.html',
             controller: 'NavbarController',
             controllerAs: 'navbar',
-            bindToController: true
+            bindToController: true,
+            scope: {
+                admin: '@',
```

Comment:
Yeah, there are nuances to be worked out in how the navbar behaves and what it shows when.  The parameter isn't intended so much as "is there an admin user logged in" as "is this an admin page".  So maybe the `$state` service rather than `AuthService` for that part.  Though showing a login link for users who aren't logged in and a link to the admin views for users who are might be a sensible thing to do, and would probably hang on `AuthService`.